### PR TITLE
Tomasz

### PR DESCRIPTION
**Pull request by TGacek89**

### CODE REVIEW

**Review by VasylPosmit — APPROVED** [review]

**Review by celpjmnz — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+
+function divNums(...nums){
+    let res = nums.reduce((num1, num2) => num1 / num2)
+    return res
+}
+
+function multiNums(...nums){
+    let res = nums.reduce((num1, num2) => num1 * num2)
+    return res
+}
```
Comment (VasylPosmit) [inline index.js:61]: looks good! Let's merge it!

File: index.js [hunk 2]
```diff
@@ -1,6 +1,63 @@
...
+
+function divNums(...nums){
+    let res = nums.reduce((num1, num2) => num1 / num2)
+    return res
+}
+
+function multiNums(...nums){
+    let res = nums.reduce((num1, num2) => num1 * num2)
```
Comment (VladimirGOU) [inline index.js:59]: Ok

File: index.js [hunk 3]
```diff
@@ -1,6 +1,63 @@
...
+const number2 = parseFloat(prompt('Enter second number: '));
+
+switch(operator) {
+    case '+':
+         result = number1 + number2;
+        console.log(`${number1} + ${number2} = ${result}`);
+        break;
+
+    case '-':
+         result = number1 - number2;
```
Comment (VasylPosmit) [inline index.js:22]: MINOR why do we have two different implementation of the same functionality?

Comment (VladimirGOU) [reply index.js:22]: lets ask Tomas)

**Review by VladimirGOU — APPROVED** [review]

**Pull request merged**